src/app/api/artifacts/dr-sample-cvr.ts

```diff
@@ -1,149 +1,21 @@
-export const DR_SAMPLE_CVR = `<?xml version="1.0" encoding="UTF-8"?>
-<CastVoteRecordReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://itl.nist.gov/ns/voting/1500-103/v1 ../../../../../CastVoteRecords/NIST_V0_cast_vote_records.xsd"
-    xmlns="http://itl.nist.gov/ns/voting/1500-103/v1">
+export const DR_SAMPLE_CVR = `
```

Comment:
What was the reason for this change? Can it be reverted?

Reply:
We are using a new CVR determined by the new EDFs. We are using it for integration tests with the `Provisioner`. Is there an issue with changing the CVR?

Reply:
Great question - @awoflaherty what do you think?

Reply:
contents in the /artifacts/ directory will be removed as part of the Markit integration.  So while you can update Rosie with it, I will not recommend moving this directory to Markit.